test/e2e/apimachinery/webhook.go

```diff
@@ -115,19 +115,37 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		cleanWebhookTest(client, namespaceName)
 	})
 
-	ginkgo.It("Should be able to deny pod and configmap creation", func() {
+	/*
+		Release : v1.16
+		Testname: Admission webhook, deny create
+		Description: Register an admission webhook configuration that denies configmap and pod creation and ensure
+		creation attempts of both are denied.
+	*/
+	framework.ConformanceIt("Should be able to deny pod and configmap creation", func() {
```

Comment:
nit: lowercase "should" (applies to all tests here)

Reply:
All test names updated. I'll do this for the CRD Conformance PR as well.